includes/AI/wpuf-ai-system-prompt-compact.md

```
@@ -0,0 +1,237 @@
+  "insert_image": "yes",
+  "width": "large",
+  "restriction_to": "max",
+  "restriction_type": "character",
+  "text_editor_control": [],
+  "wpuf_cond": {"condition_status":"no","cond_field":[],"cond_operator":["="],"cond_option":["- Select -"],"cond_logic":"all"},
+  "wpuf_visibility": {"selected":"everyone","choices":[]}
+}
+```
+
+## FIELD TYPE MAPPING (CRITICAL)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)